libs/core/src/components/sage-tab/sage-tab.tsx

```diff
@@ -0,0 +1,84 @@
+import { Component, Element, Host, h, Prop, Event, EventEmitter, State } from '@stencil/core';
+
+@Component({
+  tag: 'sage-tab',
+  styleUrl: 'sage-tab.scss',
+  shadow: false,
+})
+export class SageTab {
+  @Element() el: HTMLElement;
+
+  /**
+   * Sets the related tab name, this name must match a `sage-tabpanel`'s tab property, required
+  */
+  @Prop() tab: string;
+
+  /**
+   * Keeps track of the activeTab, this property is passed in by parent component
+  */
+  @Prop({mutable: true}) activeTab: string;
+
+  /**
+   * Keeps track of the parentComponent unique id, this property is passed in by parent component
+  */
+  @Prop({mutable: true}) parentComponent: string;
+
+  /**
+   * Keeps track of if the expected tab variant, this property is passed in by parent component
+  */
+  @Prop() variant: string;
+
+  /**
+   * Keeps track of if the tabpanel is selected, this property is computed on `componentWillUpdate()`
+  */
+  @State() selected = false;
+ 
+  /**
+   * Emits an event upon tab click for `sage-tab` and `sage-tabpanel` to listen for
+   */
+  @Event() tabClick: EventEmitter<object>;
+  private onTabClick(tab, parentComponent) {
+    this.activeTab = tab;
+    this.tabClick.emit([tab, parentComponent]);
+  }
+
+  private matchActiveTab() {
+    if (this.activeTab && this.activeTab === this.tab) {
```

Comment:
Regarding `this.activeTab && this.activeTab` is that part of the conditional needed as it seems like it can go? Would it work the same as `this.activeTab === this.tab`?